app/controllers/homes_controller.rb

```diff
@@ -1,6 +1,20 @@
 class HomesController < ApplicationController
-  # feel free to replace this controller
 
   def index
   end
+
+  def authenticated
+
+  end
+
+  def authorized
+
+  end
+
+  protected
+
+  def authorize_user
+
+  end
```

Comment:
ummm....Im seeing these, but not seeing how they are used??

Reply:
they arent used at all it seems. delete unless being actively used and completed